lbry/lbry/stream/descriptor.py

```diff
@@ -46,6 +47,29 @@ def file_reader(file_path: str):
             offset += bytes_to_read
 
 
+def sanitize_file_name(dirty_name: str):
+    exprs = [
+        '[<>:"/\\\|\?\*]+',                         # Illegal characters
+        '[\\x00-\\x1F]+',                           # All characters in range 0-31
+        '[ \t]*(\.)+[ \t]*$',                       # Dots at the end
+        '(^[ \t]+|[ \t]+$)',                        # Leading and trailing whitespace
+        '^CON$', '^PRN$', '^AUX$',                  # Illegal names
+        '^NUL$', '^COM[1-9]$', '^LPT[1-9]$']
+
+    RES = re.compile('(' + '|'.join((expr for expr in exprs)) + ')')
```

Comment:
This makes it seem like a constant but it actually gets recompiled every time `sanitize_file_name` is called. Perhaps move this out of the function so that the compilation only happens once?